### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by atul9452**
Please Review it.

### CODE REVIEW

File: Sorting_Module/quicksort.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+"""------------Quick Sorting--------------------
+
+It performs quick sorting on a input list where input list may contain integer or float.It is also based on divide and
+conquer algorithm.
+
+Attributes:
+    No module level attributes are declared in this module.
+
+Todo::
+    Using more sorting algorithm we have to create package with folder of all the module
```
Comment (nitin949) [inline Sorting_Module/quicksort.py]: what is this ?

File: Sorting_Module/quicksort.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+"""
+
+
+def quick_sort(l_input):
+    """his method is passing the input list to quick function to  be processed.
```
Comment (nitin949) [inline Sorting_Module/quicksort.py]: his or this ?

File: Sorting_Module/sorting.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    @func_execution_time
+    def quick_sort(self):
+        """This module will be called for Quick Sorting"""
+        sorted_list = quicksort.quick_sort(self.l_input)
+        return sorted_list
+
+
+if __name__ == '__main__':
+    # unsorted_list = [1, 3, 2, 4, 7, 9]
+    obj = Sort(['w', 'c', '1']).selection_sort()
```
Comment (nitin949) [inline Sorting_Module/sorting.py]: Is your program working for strings ?

**Comment by atul9452** [comment]
added logging file and testcase

**Pull request closed without merging**